h1st/model/ml_modeler.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any, Dict
+from h1st.model.modeler import Modeler
+from h1st.model.ml_model import MLModel
+
+
+class MLModeler(Modeler):
+    """
+    Base class for H1st ML Modeler.
+    """
+    @property
+    def model_class(self) -> Any:
+        return getattr(self, "__model_class", None)
+
+    @model_class.setter
+    def model_class(self, value):
+        setattr(self, "__model_class", value)
+    
+    def build(self) -> MLModel:
+        """
+        Implement logic to create the corresponding MLModel object.
+        """
+        data = self.load_data()
+        training_data = self.generate_training_data(data)
```

Comment:
One suggestion from CTN was changing generate_training_data to get_training_data to make it more general. For example, there are some cases when users already have completed training data. 
However, we already have a load_data function, so I think we don't need get_training_data as well. 
In conclusion, I think we can remove generate_training_data step. What do you think ?